### ISSUE TITLE
[CLI] Bad State: Client Closed

### ISSUE THREAD

**Issue by FilledStacks** [Stacked-Org/stacked#869]
When creating a new app I see the following error at the end.

```shell
Unhandled exception:
Bad state: Client is closed
#0      _HttpClient._openUrl (dart:_http/http_impl.dart:2683:7)
#1      _HttpClient.postUrl (dart:_http/http_impl.dart:2601:49)
#2      IOPostHandler.sendPost (package:usage/src/usage_impl_io.dart:116:31)
#3      AnalyticsImpl._trySendBatches (package:usage/src/usage_impl.dart:316:36)
#4      AnalyticsImpl._enqueuePayload (package:usage/src/usage_impl.dart:277:7)
#5      AnalyticsImpl.sendEvent (package:usage/src/usage_impl.dart:153:12)
#6      AnalyticsService.logExceptionEvent (package:stacked_cli/src/services/analytics_service.dart:154:22)
<asynchronous suspension>
```

**Comment by FilledStacks** [comment]
@ferrarafer when you get some time please look into this.

**Comment by markbulingit** [comment]
in my case, the error started when I answered [yes] to user-improvement program

**Comment by FilledStacks** [comment]
@markbulingit thanks for adding that information. We'll get on this soon.

**Comment by devRaphe** [comment]
I experienced this too, when I answered yes to the user-improvement program.

**Comment by FilledStacks** [comment]
@ferrarafer has this been looked at?

**Comment by ferrarafer** [comment]
Never happened to me, I will try to reproduce it today and see what I find.

**Comment by ferrarafer** [comment]
I tried to create a new app with stacked `cli` with and without analytics **successfully**.

I will try the `cli` on a new operative system asap because on my current system I don't get the prompting of user-improvement.

**Comment by FilledStacks** [comment]
Fix for this is out in `1.5.0`
